cernopendata/cold_storage/service.py

```diff
@@ -137,19 +137,22 @@ def check_submitted():
                     info = manager.doOperation(
                         action,
                         transfer.record_id,
-                        limit=limit - submitted,
+                        limit=None,
                         register=True,
                         force=False,
                         dry=False,
+                        max_transfers=max_transfers - submitted,
                     )
                     logger.debug(f"Got {info}")
                     if info:
                         submitted += len(info)
                         transfer.num_files += len(info)
                         transfer.size += sum(item.size for item in info)
                     transfer.started_at = datetime.utcnow()
-                    logger.info(f"THE LIMIT WAS {limit}, AND WE SUBMITTED {submitted}")
-                    if limit == submitted:
+                    logger.info(
+                        f"THE LIMIT WAS {max_transfers}, AND WE SUBMITTED {submitted}"
+                    )
+                    if max_transfers == submitted:
```

Comment:
question: are we sure that submitted cannot go above max_transfers? In other words,:
```suggestion
                    if max_transfers <= submitted:
```